project-base/storefront/components/Blocks/BlogPreview/BlogPreviewPlaceholder.tsx

```diff
@@ -0,0 +1,73 @@
+import { BlogPreviewProps } from './BlogPreview';
+import { ExtendedNextLink } from 'components/Basic/ExtendedNextLink/ExtendedNextLink';
+import useTranslation from 'next-translate/useTranslation';
+
+type BlogPreviewPlaceholderProps = Pick<BlogPreviewProps, 'blogArticles' | 'blogUrl'>;
+
+export const BlogPreviewPlaceholder: FC<BlogPreviewPlaceholderProps> = ({ blogArticles, blogUrl }) => {
+    const { t } = useTranslation();
+
+    return (
+        <div>
+            <h2 className=" text-creamWhite">{t('Shopsys magazine')}</h2>
```

Comment:
There is unneeded space before class name.

Reply:
Thanks, removed